test/unit/trace.test.js

```diff
@@ -758,6 +773,38 @@ tap.test('should set URI to /Unknown when URL is not known/set on transaction',
   t.end()
 })
 
+tap.test('should obfuscate URI using regex when pattern is set', async (t) => {
+  const URL = '/abc/123/def'
```

Comment:
since this is using the greedy flag, can you add an extra number path segment? right now the greedy isn't really doing anything.